### PR TITLE
:seedling: Modify reviewers

### PR DESCRIPTION
**Pull request by Rozzii**
Michael is not working on the project currently thus others volunteered
to be reviewers as there was nobody who would have had 10 or more
commits and isn't already an approver.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -6,8 +6,13 @@ approvers:
  - furkatgofurov7
 
 reviewers:
- - macaptain
+ - smoshiur1237
+ - Rozzi
```
Comment (furkatgofurov7) [inline OWNERS:10]: This is your own GitHub handle @Rozzii :grinning:
```suggestion
 - Rozzii
```

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 1 above)
Comment (Rozzii) [reply OWNERS]: Thanks!

**Review by furkatgofurov7 — APPROVED** [review]
@Rozzii this looks good, thanks! It would be nice to sync owners' file between `main` branch and release branches (`release-0.1` branch in case of IPAM repo)l. Also, if you are willing to, it would be great to check the CAPM3 repo and if needed sync main with the `release-0.5` branch. 

/approve

**Pull request merged**